modules/mono/editor/bindings_generator.h

```diff
@@ -31,16 +31,17 @@
 #ifndef BINDINGS_GENERATOR_H
 #define BINDINGS_GENERATOR_H
 
+#include "core/typedefs.h" // DEBUG_METHODS_ENABLED
+
+#if defined(DEBUG_METHODS_ENABLED) && defined(TOOLS_ENABLED)
+
 #include "core/doc_data.h"
 #include "core/object/class_db.h"
 #include "core/string/string_builder.h"
+#include "core/string/ustring.h"
 #include "editor/doc_tools.h"
 #include "editor/editor_help.h"
 
-#if defined(DEBUG_METHODS_ENABLED) && defined(TOOLS_ENABLED)
-
-#include "core/string/ustring.h"
```

Comment:
There was a bunch of defines before the `#if` guards, seemingly without reason, but upon moving them inside the guards it started errorring at linking time. Turns out `DEBUG_METHODS_ENABLED` is defined in `typedefs.h` so this needs to be included first.